modules/edu.gemini.seqexec.model/src/test/scala/edu/gemini/seqexec/model/BoopicklingSpec.scala

```diff
@@ -17,7 +17,8 @@ object SharedModelArbitraries {
   def sequencesQueueArb[A](implicit arb: Arbitrary[A]): Arbitrary[SequencesQueue[A]] = Arbitrary {
     for {
       b <- Gen.listOfN[A](maxListSize, arb.arbitrary)
-    } yield SequencesQueue(Conditions.default, b)
+      o <- Gen.option(Gen.alphaStr)
+    } yield SequencesQueue(Conditions.default, o, b)
```

Comment:
Agree. I didn't see changes on the time on my local machine in terms of time but yeah, travis keeps failing